dataeng/jobs/analytics/WarehouseTransforms.groovy

```diff
@@ -23,6 +23,7 @@ class WarehouseTransforms{
                     stringParam('DBT_PROFILE', env_config.get('DBT_PROFILE', allVars.get('DBT_PROFILE')), 'dbt profile from analytics-secure to work on.')
                     stringParam('DBT_TARGET', env_config.get('DBT_TARGET', allVars.get('DBT_TARGET')), 'dbt target from analytics-secure to work on.')
                     stringParam('DBT_COMMAND', env_config.get('DBT_COMMAND', allVars.get('DBT_COMMAND', 'run')), 'dbt command to be executed.  Default is \'run\'.')
+                    stringParam('SEED_SELECTOR', env_config.get('SEED_SELECTOR', allVars.get('SEED_SELECTOR', '\\*')), 'Model selector that will be used when running the seed step.  Defaults to "\\*".')
```

Comment:
It looks like you changed this already, but when it was still `'"*"'`, I wrote this:

Actually, are you sure you want double quotes inside the single quotes?  I assume that makes the text box in the web UI contain `"*"` by default, but that exposes the low-level machinery of the bash.

Reply:
but, it looks like you're trying different escaping sequences.  If you find that still isn't working, I think just double quoting the variable in the bash script is the way to go.